webxr/openxr/input.rs

```diff
@@ -147,18 +153,104 @@ impl OpenXRInput {
             )
             .unwrap();
 
-        let hand_tracker = if needs_hands {
+        let (_hand_tracker, joints) = if needs_hands {
```

Comment:
What's the purpose of ignoring this value?

Reply:
We need to hold on to it in the struct but it generates an unused error. I'll `allow()` it